source/Ship.cpp

```diff
@@ -3324,7 +3324,11 @@ int Ship::Crew() const
 	return crew;
 }
 
-
+// Calculate drag, accounting for drag reduction.
```

Comment:
The empty lines count is incorrect here and after this function (always 3 empty lines between member function definitions).